src/components/footer/Footer.jsx

```diff
@@ -3,46 +3,45 @@ import './Footer.css';
 
 function Footer() {
   return (
-    <div className="footer">
+    <div className="footer fixed bottom-0 left-0">
+
       <div className="footer-container">
+
         <p>
           © 2023 All rights reserved
         </p>
-        <div className="flex gap-2">
+
+        <div className="flex items-center">
+
+          <div className="pr-2">
+            <a href="https://github.com/chingu-voyages/v46-tier2-team-20" target="blank"><img src="/github-mark-white.png" alt="github-logo" width="30" height="30" /></a>
+          </div>
+
           <div>
-            <p>Designer:</p>
             <p>Developers:</p>
+            <p>Designer:</p>
           </div>
+
           <div>
-            <span>
-              <a href="https://github.com/nghia-nguyen-dev" target="blank">
-                Nghia
-              </a>
-            </span>
             <div>
-              <span>
-                <a href="https://github.com/AOA19">Alexandra</a>
-              </span>
+              <a href="https://github.com/AOA19">Alexandra</a>
               {' '}
-              <span>
-                <a href="https://github.com/allison-truhlar" target="blank">Allison</a>
-              </span>
+              <a href="https://github.com/allison-truhlar" target="blank">Allison</a>
               {' '}
-              <span>
-                <a href="https://github.com/Emimint" target="blank">Emimint</a>
-              </span>
+              <a href="https://github.com/Emimint" target="blank">Emimint</a>
               {' '}
-              <span>
-                <a href="https://github.com/komalgill0310" target="blank">Komal</a>
-              </span>
+              <a href="https://github.com/komalgill0310" target="blank">Komal</a>
```

Comment:
The value for the `target` attribute should be `_blank`. Right now the when you click on each of our names it doesn't open in a new tab.